packages/client-twitter/src/base.ts

```diff
@@ -81,6 +81,19 @@ class RequestQueue {
     }
 }
 
+
+export function getScrapper(username:string):twitterClient {
```

Comment:
should this be renamed to "getTwitterClient"?

Reply:
there's about 3 things already called client here and only one scrapper, so it was the more specific term that had meaning to me

Reply:
Is this a supposed to be "scraper" then? as in webscraping?

Reply:
```
import {
    QueryTweetsResponse,
    Scraper,
    SearchMode,
    Tweet,
} from "agent-twitter-client";
```

That's the technical internal name for this specific thing